algosdk/v2client/algod.py

```diff
@@ -719,7 +719,7 @@ def set_timestamp_offset(
         return self.algod_request("POST", req, **kwargs)
 
     def get_ledger_state_delta(
-        self, round: int, **kwargs: Any
+        self, round: int, response_format: str = "json", **kwargs: Any
```

Comment:
It'd be nice to add the new argument in the docstrings below (args section) as well